test_files/type_propaccess.no_externs/clutzed_namespace.d.ts

```diff
@@ -0,0 +1,11 @@
+declare namespace ಠ_ಠ.clutz.type_propaccess.nested.clazz {
```

Comment:
basic question - why is tsickle consuming clutz .d.ts file? Those are supposed to have a source .js, closure should be consuming those and not the clutz->tsickle abomination.

Reply:
This simulates code referencing clutz processed TS, but the clutz code does not go through tickle.